docs/reference/buildx_history_trace.md

```diff
@@ -15,3 +15,55 @@ Show the OpenTelemetry trace of a build record
 
 <!---MARKER_GEN_END-->
 
+## Description
+
+View the OpenTelemetry trace for a completed build. This command loads the
+trace into a Jaeger UI viewer and opens it in your browser.
+
+This helps analyze build performance, step timing, and internal execution flows.
+
+## Examples
+
+### <a name="open-opentelemetry-trace"></a> Open the OpenTelemetry trace for the most recent build
+
+This command starts a temporary Jaeger UI server and opens your default browser
+to view the trace.
+
+```console
+docker buildx history trace
+```
+
+### <a name="open-trace-specific-build"></a> Open the trace for a specific build
+
+```console
+# Using a build ID
+docker buildx history trace qu2gsuo8ejqrwdfii23xkkckt
+
+# Or using a relative offset
+docker buildx history trace ^1
+```
+
+### <a name="run-Jaegar-UI-port"></a> Run the Jaeger UI on a specific port
+
+```console
+# Using a build ID
+docker buildx history trace qu2gsuo8ejqrwdfii23xkkckt --addr 127.0.0.1:16686
+
+# Or using a relative offset
+docker buildx history trace ^1 --addr 127.0.0.1:16686
+```
+
+### <a name="compare-traces"></a> Compare two build traces
+
+Compare two specific builds by name:
+
+```console
+# Using build IDs
+docker buildx history trace --compare qu2gsuo8ejqrwdfii23xkkckt qsiifiuf1ad9pa9qvppc0z1l3
```

Comment:
Maybe when showing these as `--compare=qu2gsuo8ejqrwdfii23xkkckt`, `--compare=^1` it makes it clearer that this is a value for the flag.